File: ruby/hello_world/hello_world.rb
require 'asana'

# replace with your personal access token. 
personal_access_token = '0/123456789....'

# Construct an Asana client
client = Asana::Client.new do |c|
  c.authentication :access_token, personal_access_token
end

# Get your user info
me = client.users.me
# Print out your information
puts "Hello world! " + "My name is " + me.name + " and I my primary Asana workspace is " + me.workspaces[0].name + "."

File: javascript/hello_world/hello_world.js
var asana = require('asana');

// replace with your personal access token.
var personalAccessToken = '0/123456789....';

// construct an Asana client
var client = asana.Client.create().useAccessToken(personalAccessToken);

// Get your user info
client.users.me()
  .then(function(me) {
    // Print out your information
    console.log('Hello world! ' + 'My name is ' + me.name + '.');
});
